parsing.go

```diff
@@ -283,21 +314,164 @@ func parseReq(txt string) (*Req, error) {
 		return nil, fmt.Errorf("Requirement body must not be empty: %s", r.ID)
 	}
 
+	// PARENTS must be punctuation/space separated list of parseable req-ids.
+	err = parseParents(r)
+	if err != nil {
+		return nil, err
+	}
+
+	return r, nil
+}
+
+// parseReqTable reads a table of requirements one row at a time and parses the content into Req structures which are
+// then returned in a slice.
+//
+// Tables have the following format:
+// | ID | Title | Body | Attribute1 | Attribute2 |
+// | --- | --- | --- | --- | --- |
+// | ReqID | <text> | <text> | <text> | <text> |
+//
+// The first column must be "ID" and each row must contain a valid ReqID. Other columns are optional.
+//
+// @llr REQ-TRAQ-SWL-25
+func parseReqTable(txt string, reqs []*Req) ([]*Req, error) {
+
+	var attributes []string
+
+	// Split the table into rows and loop through
+	for rowN, rowS := range strings.Split(txt, "\n") {
+
+		// The first row contains the attribute names for each column, the first column must be "ID"
+		if rowN == 0 {
+			if reTableHeader.MatchString(rowS) {
+				attributes = splitTableLine(rowS)
+				for i, a := range attributes {
+					attributes[i] = strings.ToUpper(a)
+				}
+			} else {
+				return reqs, fmt.Errorf("first column must be \"ID\"")
```

Comment:
is this the only failure mode for the regular expression. If not, it would be good to try to mention other issues.

Reply:
True, the main other failure case is having a table with only one column. Updated message and added test.